lib/mobile_app_backend/predictions/pub_sub.ex

```diff
@@ -45,22 +45,21 @@ defmodule MobileAppBackend.Predictions.PubSub do
 
   Based on https://github.com/mbta/dotcom/blob/main/lib/predictions/pub_sub.ex
   """
-  use GenServer
+  use MobileAppBackend.PubSub,
+    broadcast_interval_ms:
+      Application.compile_env(:mobile_app_backend, :predictions_broadcast_interval_ms, 10_000),
+    broadcast_message_name: :new_vehicles
```

Comment:
`broadcast_message_name` doesn't appear to be used within `PubSub.__using__/1`, so it doesn't need to be specified here and in `Vehicles.PubSub` (plus this is the wrong value anyway).

Reply:
Ah yep that is a holdover from trying to stuff more pieces into the macro. Will remove!

Reply:
Ah yep that is a holdover from trying to stuff more pieces into the macro. Will remove!